tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -112,6 +129,18 @@
     default=None,
     help="Modality of the multimodal requests.",
 )
+@optgroup.option(
+    "--image_data_format",
```

Comment:
Question: do models usually support more than one format? If not, can we infer this from somewhere instead of requiring the user to specify it?